config/manager/manager.yaml

```diff
@@ -21,6 +21,7 @@ spec:
         - "--metrics-bind-addr=127.0.0.1:8080"
         - "--leader-elect"
         - "--feature-gates=EKS=${CAPA_EKS:=true},EKSEnableIAM=${CAPA_EKS_IAM:=false},EKSAllowAddRoles=${CAPA_EKS_ADD_ROLES:=false},EKSFargate=${EXP_EKS_FARGATE:=false},MachinePool=${EXP_MACHINE_POOL:=false},EventBridgeInstanceState=${EVENT_BRIDGE_INSTANCE_STATE:=false},AutoControllerIdentityCreator=${AUTO_CONTROLLER_IDENTITY_CREATOR:=true}"
+        - "--v=${EKS_CP_LOGLEVEL:=4}"
```

Comment:
Now we have merged the controllers together, it's not specific to EKS. 

perhaps we should change its name to `CAPA_LOGLEVEL`, wdyt?

Reply:
sounds fine, will make the change